Dockerfile.caasp-1_0

```diff
@@ -0,0 +1,74 @@
+FROM opensuse:42.2
+RUN zypper ar -f http://download.opensuse.org/repositories/YaST:/SLE-12:/SP2/SLE_12_SP2/ yast_sle_12_sp2
```

Comment:
I'd raise the priory of the repository to prefer the packages from `yast_sle_12_sp2` regardless the version numbers in original 42.2 repo, to avoid possible problems with version numbers.

See https://github.com/yast/docker-yast-ruby/compare/sp2_test?expand=1#diff-c26a7079239c13bb2d3ab89ae93b1b93R9